### PR TITLE
#184651051 Fix husky configuration to stage untracked files after formatting

### PR DESCRIPTION
**Pull request by Kress2000**
What does this PR do?
Checks code syntax and formats the warnings ad errors after then auto add and commit the changes before pushing

Description of Task to be completed?
installs husky, and initialize eslint
configures again esltrc.json and pre-commit file
More details here on the pictures:
- error example: 
image
- If it is fixable with
```
lint:fix
```
 or
``` 
prettier:fix
```
 commands,  this will be formated automatically by 
```
git commit -m "your message"
```
after staging your work by
```
git add .
```
 _details here on the pictures_
- error solved: 
image

Any background context you want to provide?
--

[Delivers #184651051]

### CODE REVIEW

**Review by munezerobagira — APPROVED** [review]

**Comment by Kress2000** [comment]
> check comment

solved

File: src/database/models/usertest.js [hunk 1]
```diff
@@ -1,23 +1,29 @@
 const { Model } = require('sequelize');
 
+/* eslint-disable*/
 module.exports = (sequelize, DataTypes) => {
```
Comment (munezerobagira) [inline src/database/models/usertest.js:4]:
```suggestion
export default (sequelize, DataTypes) => {
```

File: src/tests/model.test.js [hunk 2]
```diff
@@ -14,26 +14,26 @@ const sequelize = new Sequelize(config.url);
...
+    allowNull: false,
   },
   email: {
     type: Sequelize.STRING,
     allowNull: false,
-    unique: true
-  }
+    unique: true,
+  },
 });
```
Comment (munezerobagira) [inline src/tests/model.test.js:28]:
```suggestion
```

File: src/tests/model.test.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ const assert = require('assert');
 const db = require('../database/models');
 
 const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../database/config/config.js')[env];
+const config = require(`${__dirname}/../database/config/config.js`)[env]; //eslint-disable-line
 
 const { Usertest } = db;
 const sequelize = new Sequelize(config.url);
```
Comment (munezerobagira) [inline src/tests/model.test.js:10]:
```suggestion
import Userschema from '../database/models/usertest.ts'
```

**Review by munezerobagira — CHANGES_REQUESTED** [review]

**Review by munezerobagira — CHANGES_REQUESTED** [review]
delete `build` folder and it to the .gitnore

**Comment by Kress2000** [comment]
image
By trying to squash the commits: there are left 3 three commits and can't be less unless I reset the changes to develop, otherwise, I am getting conflicts

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**